solr_schema.xml

```diff
@@ -133,7 +136,7 @@
 
     <field name="_version_" type="string" indexed="true" stored="true"/>
 
-    <dynamicField name="*_date" type="date" indexed="true" stored="true" multiValued="false"/>
+    <!--<dynamicField name="*_date" type="date" indexed="true" stored="true" multiValued="false"/>-->
```

Comment:
revert the changes to this file, this is the solr schema for ckan not for ATI/PD search